App1/App1/layout-large.xml

```diff
@@ -41,9 +46,9 @@
 
     <!-- Taskbar / layers -->
     <layer-list id="layers"/>
-
-    <!-- Taskbar / legend -->
-    <legend id="legend"/>
+    
+    <!-- Basmeap picker -->
+    <basemap-picker/>
```

Comment:
Intentional removal of `legend` i assume?

Reply:
It's not gone, just moved into an expand on the map.